packages/grafana-schema/src/schema/dashboard/v2alpha0/dashboard.schema.cue

```diff
@@ -576,6 +576,25 @@ ResponsiveGridLayoutItemSpec: {
   element: ElementReference
 }
 
+TabsLayoutKind: {
+  kind: "TabsLayout"
+  spec: TabsLayoutSpec
+}
+
+TabsLayoutSpec: {
+  tabs: [...TabsLayoutTabKind]
```

Comment:
Nit: maybe we should unify the naming to be `items` across diferent layout types?

Reply:
I agree. This is why I suggested to also standardized the kinds under `<Foo>LayoutItem` for `<Foo>Layout`

Reply:
Yes maybe we can have a discussion around this. The main reason I didn't want to go with item is because of the difference between a layout that contain a layout and a layout that contains GridItems.

Rows and Tabs layouts are layouts that cannot contain GridItems (panels), they must contain a child layout that holds the items.

I'm not married to the naming scheme, so happy to change it if we agree on everything being items :)